server/src/routines/game.ts

```diff
@@ -245,16 +245,48 @@ function getAvailableActions(
 
 				if (pickableSlots.length === 0) return reducer
 
-				if (card.isHealth() && !reducer.includes('PLAY_HERMIT_CARD')) {
+				if (
+					card.isHealth() &&
+					!reducer.includes('PLAY_HERMIT_CARD') &&
+					game.components.find(
+						SlotComponent,
+						card.props.attachCondition,
+						query.slot.hermit,
+					)
+				) {
 					reducer.push('PLAY_HERMIT_CARD')
 				}
-				if (card.isAttach() && !reducer.includes('PLAY_EFFECT_CARD')) {
+				if (
+					card.isAttach() &&
+					!reducer.includes('PLAY_EFFECT_CARD') &&
+					game.components.find(
+						SlotComponent,
+						card.props.attachCondition,
+						query.slot.attach,
```

Comment:
The unit test fails because String is an attach card that can be placed in an item slot (even when the opponent has no available attach slots)
```suggestion
						query.some(query.slot.attach, query.slot.item),
```

Reply:
Thanks for finding the bug! I don't like the solution so I am going to try something else.

Reply:
The solution in the PR right now passes all unit tests and fixes the fuzz testing errors.